index.js

```diff
@@ -1,13 +1,11 @@
 import React, { Component } from 'react';
```

Comment:
are you sure ? did you try it without it ? i think the import react tweaks some bits in the import/export process.